android/app/src/main/java/com/microsoft/codepush/common/managers/SettingsManager.java

```diff
@@ -137,7 +137,7 @@ public boolean existsFailedUpdate(String packageHash) throws CodePushMalformedDa
      */
     public boolean isPendingUpdate(String packageHash) throws CodePushMalformedDataException {
         CodePushPendingUpdate pendingUpdate = getPendingUpdate();
-        return pendingUpdate != null && pendingUpdate.isPendingUpdateLoading() &&
+        return pendingUpdate != null && !pendingUpdate.isPendingUpdateLoading() &&
```

Comment:
Please double check this condition.

Reply:
@sergey-akhalkov no need - that was the bug I fixed.